### DISCUSSION TITLE
Making a clean reinstall on win10

### DISCUSSION THREAD

**Discussion by luppahannu** [Xpra-org/xpra#3890] [Q&A]
Hello!

I have been using Xpra for a while now to run programs from my linux machine on my win10 laptop. A little while ago things started to break down. "Mode 'seamless' not found in command line arguments" error started to pop up. I tried installing the latest version, didn't help. I tried uninstalling and the deleting the xpra folders from my user/AppData/local and roaming. Reinstalled, didn't work. After some investigating I found out that there's a Xpra folder under C:\ProgramData\, deleted that between uninstall and reinstall and things got even worse.

Trying to start from the GUI is totally broken, trying to start a session opens up an error window with a long message I cannot select as text. There seems to be a UnicodeDecodeError in run_opengl_probe... I try to start with no OpenGL, but none of the options I set seem to stick. I have a Finnish keyboard but Indonesian is always selected when I start the GUI. 

I can start a somewhat working connection with powershell, but when I do something like try to open a file, and a new window opens, the "Mode 'seamless' not found" error pops up and the connection closes.

Is there a way I can make an actual clean install of xpra on win10? Deleting the mentioned folders did not help. What else is there to wipe out?

The version I'm trying to install is 4.4.5 (server xpra is version v4.4.4-r0).

**Comment by totaam** [comment]
There is an issue with opengl on 4.4.5, if that causes you problems then you may want to use 4.4.4 or the latest 5.0 beta builds.

> I found out that there's a Xpra folder under `C:\ProgramData`

That's where the software's default configuration files are kept. Those are version specific so they are managed by the installer.
I don't see how these would cause problems.
You can run the `Path_info.exe` tool to see the various paths that xpra will use for location configuration files.

> There seems to be a `UnicodeDecodeError` in `run_opengl_probe`...

Please report the full error, as a screenshot if needed.

> I have a Finnish keyboard but Indonesian is always selected when I start the GUI.

Without the client output, that's impossible to debug.
Try running (ie: with ssh here):
`Xpra_cmd.exe -d keyboard attach ssh://user@host/`

> I can start a somewhat working connection with powershell,

The shell should not make any difference, unless the environment is somewhat corrupted in one of them.

> when I do something like try to open a file

How? What application?

> Is there a way I can make an actual clean install of xpra on win10?

Running the installer is all that is needed.

As for everything else, there aren't enough details for me to try diagnose. Please see:
https://github.com/Xpra-org/xpra/wiki/Reporting-Bugs

**Comment by totaam** [comment]
Not heard back.
